doc/api/Buffer_Control/getMaxBufferSize.md

```diff
@@ -0,0 +1,19 @@
+# getMaxBufferSize
+
+## Description
+
+Returns the maximum buffer memory size limit , in kilobytes.
+
+This setting can be updated either by:
+
+- calling the `setMaxBufferSize` method.
```

Comment:
you can add links to those two (if you don't know how, it's like a regular markdown link to the other markdown, for `maxBufferSize`, you may want to add an anchor with `#`, you can look at other places in the doc to know the form of those - IIRC, it's just the name of the option in lowercase but I'm not certain).